### PR TITLE
Create my.js

### PR DESCRIPTION
**Pull request by pete-arif**
suggestion for Bahasa Melayu (Malaysia) locale

### CODE REVIEW

File: src/locale/my.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'id',
```
Comment (iamkun) [inline src/locale/my.js]: please keep name the same with file name

**Comment by iamkun** [comment]
Should this be `ms` instead of `my`? https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

**Comment by iamkun** [comment]
and file name pls

**Comment by pete-arif** [comment]
> and file name pls
> Should this be `ms` instead of `my`? https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

It should be... apologies. 👍  
_(personally, not really agree to that ISO naming for the language, but standards is standards)_

**Pull request merged**